### ISSUE TITLE
the buildx component is missing on dockerhost-skytap-ubuntu2004-ppc64le-1

### ISSUE THREAD

**Issue by sophia-guo** [adoptium/infrastructure#3234]
dockerhost-skytap-ubuntu2004-ppc64le-1 is the only one machine to build the debian installer for ppc64le. The release installer job got error message  for all jdk|jre version

```
> Task :jdk:debian:packageJdkDebian FAILED
The Value Of ARCH is: ppc64el
ERROR: BuildKit is enabled but the buildx component is missing or broken.
       Install the buildx component to build images with BuildKit:
       https://docs.docker.com/go/buildx/

FAILURE: Build failed with an exception.
```
For example
https://ci.adoptium.net/job/adoptium-packages-linux-pipeline_new/864/console

**Comment by sxa** [comment]
Is this some new requirement that we didn't have before? I don't believe we've got anything in our playbooks that would ever have set that up.

**Comment by sophia-guo** [comment]
No idea. [dockerhost-marist-ubuntu2204-s390x-1](https://ci.adoptium.net/computer/dockerhost%2Dmarist%2Dubuntu2204%2Ds390x%2D1/) has the same issue.

**Comment by sxa** [comment]
> No idea

I feel this is important to understand before we make a change - this wasn't flagged as a problem during the 21+35 release.

> If you could take it offline

Done. I may be wrong but I think you have access to take machines offline too, so once you've tested please put that one back online as soon as it is convenient if you can.

**Comment by sophia-guo** [comment]
I don't think I have the access to take the machine offline or back online.

https://ci.adoptium.net/computer/test%2Dmarist%2Dubuntu2204%2Ds390x%2D1/ run good for once
https://ci.adoptium.net/job/adoptium-packages-linux-pipeline_new/890/consoleFull

And then can't be accessed anymore even looks like no job is running on it.
https://ci.adoptium.net/job/adoptium-packages-linux-pipeline_new/894/
https://ci.adoptium.net/job/adoptium-packages-linux-pipeline_new/895/
https://ci.adoptium.net/job/adoptium-packages-linux-pipeline_new/896/

**Comment by sxa** [comment]
Hmmm 907 seems to be running on it just now.

**Comment by sophia-guo** [comment]
Yes, https://ci.adoptium.net/job/adoptium-packages-linux-pipeline_new/905/ - https://ci.adoptium.net/job/adoptium-packages-linux-pipeline_new/907/ Looks good with https://ci.adoptium.net/computer/test%2Dmarist%2Dubuntu2204%2Ds390x%2D1/ .

**Comment by sophia-guo** [comment]
> [dockerhost-marist-ubuntu2204-s390x-1](https://ci.adoptium.net/computer/dockerhost%2Dmarist%2Dubuntu2204%2Ds390x%2D1/) has the same issue

@steelhead31 Could you also check dockerhost-marist-ubuntu2204-s390x-1? Thanks!

**Comment by steelhead31** [comment]
The package updates are failing due to  a large number of old linux kernels filling up the /boot device, I'll remove some redundant ones and perform some housekeeping...

**Comment by steelhead31** [comment]
This has a different root cause on the s390x machine, as the package updates are ok on that machine. It will require some more investigation.

**Comment by steelhead31** [comment]
Ok, I believe since docker was installed on this machine, the installation instructions have changed, the docker playbooks will need updating to follow the new process.. 

`$ sudo apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin`  
 

The docker-buildx-plugin should be installed by default now, I think it either wasn't a requirement previously OR got installed by default alongside the main docker package, I've manually installed the missing plugin, and will now test it, assuming successful, I'll create a new issue to update the docker playbooks appropriately, and close this one.

**Comment by steelhead31** [comment]
Closing as fixed, new issue #3237 for playbook updates raised.
